### ISSUE TITLE
Limit number of files to upload and limit file type

### ISSUE THREAD

**Issue by boogerlad** [balderdashy/skipper#53]
This doesn't seem to be a feature of skipper. I would implement it myself if someone could give me some guidance with the architecture of this.

**Comment by JerryC8080** [comment]
i have the save problem，mark here and i wonder how to fix it

**Comment by sailsbot** [comment]
Thanks for posting, @boogerlad.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
